jwk/helper_test.go

```diff
@@ -229,7 +230,7 @@ func TestGetOrGenerateKeys(t *testing.T) {
 	t.Run("Test_Helper/Run_GetOrGenerateKeys_With_GetKeySetError", func(t *testing.T) {
```

Comment:
should test name reflect func name change?

`Run_GetOrGenerateKeys_With_GetKeySetError` -> `Run_GetOrGenerateKeySetPrivateKey_With_GetKeySetError`

Reply:
Oh yeah good call. Might do a little more refactoring here because the name doesn't feel quite right.

Reply:
Changed my mind, left the name the same and fixed the test names.